Musescore.config

```diff
@@ -0,0 +1,2 @@
+// Add predefined macros for your project here. For example:
+// #define THE_ANSWER 42
```

Comment:
I'd do a `git reset 57d3883`, redo the commit without those files, then `git push -f`
Or even do the git rest to the commit before 77d3883, at the end of the day we'd want the commits squashed anyway